### DISCUSSION TITLE
Welcome to v46-tier1-team-01 Discussions!

### DISCUSSION THREAD

**Discussion by Shree-77** [chingu-voyages/v46-tier1-team-01#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shree-77** [comment]
Guys , after completing your part in the branch please create a pull request to merge with main. I have to start working on js part

**Reply by helenclx** [reply]
Noted
